lib/usage.js

```diff
@@ -122,9 +134,10 @@ module.exports = function (yargs, y18n) {
         return acc
       }, {})
     )
+    var theWrap = getWrap()
```

Comment:
Ah okay, gotcha now. In that case, I would go one step further and just calculate `theWrap` at the top of the `.help()` method. Then there's no need to add a 2nd argument to `maxWidth()`.